public/i18n/en/translation.json

```
@@ -814,6 +830,81 @@
+      "status": "Status",
+      "enabled": "Enabled",
+      "disabled": "Disabled",
+      "connection": "Connection",
+      "connected": "Connected",
+      "not_connected": "Not connected",
+      "download": "Download",
+      "disable": "Disable",
+      "enable": "Enable",
+      "tunnel_name": "Tunnel name",
+      "public_endpoints": "Public endpoints",
+      "local_networks": "Local networks",
+      "protocol": "Protocol",
+      "compression": "Compression",
+      "digest": "Digest",
+      "cipher": "Cipher",
+      "enforce_minimum_tls_version": "Enforce a minimum TLS version",
+      "choose_network": "Choose network",
+      "add_endpoint": "Add endpoint",
+      "add_remote_address": "Add remote address",
+      "subnet": "Subnet",
+      "auto": "Auto",
+      "server_client_negotiation": "Server/client negotiation",
+      "p2p": "P2P",
+      "tcp": "TCP",
+      "udp": "UDP",
+      "local_p2p_ip": "Local P2P IP",
+      "remote_p2p_ip": "Remote P2P IP",
+      "preshared_key": "Pre-shared Key",
+      "remote_port": "Remote port",
+      "delete_tunnel": "Delete tunnel",
+      "delete_tunnel_message": "You are about to delete tunnel '{name}'",
+      "download_tunnel_message": "Choose which type of file you want to download for the '{name}' tunnel",
+      "nethsecurity_client_configuration": "Nethsecurity client configuration",
+      "nethsecurity_client_configuration_description": "To establish a tunnel with a Nethsecurity client (ready for import)",
+      "private_key_tunnel_ca_certificates": "Private key, tunnel and CA certificates",
+      "private_key_tunnel_ca_certificates_description": "To create an OpenVPN tunnel with a third party firewall",
+      "import_configuration": "Import configuration",
+      "import": "Import",
+      "authentication": "Authentication",
+      "username": "Username",
+      "password": "Password",
+      "certificate": "Certificate",
+      "username_password_certificate": "Username, password and certificate",
+      "mode": "Mode",
+      "bridged": "Bridged",
+      "routed": "Routed",
+      "could_not_parse_configuration_file": "Could not parse configuration file",
+      "public_endpoints_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse gravida, nibh vel finibus suscipit, mauris erat porta",
+      "local_networks_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse gravida, nibh vel finibus suscipit, mauris erat porta",
+      "remote_networks_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse gravida, nibh vel finibus suscipit, mauris erat porta",
+      "extra_remote_networks_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse gravida, nibh vel finibus suscipit, mauris erat porta",
+      "topology_tooltip": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse gravida, nibh vel finibus suscipit, mauris erat porta"
```

Is it possible to format this with a little bit of html code?
```suggestion
      "topology_tooltip": "Subnet (recommended): This method offers a straightforward configuration approach. Instead of using a point-to-point setup, utilize a subnet configuration by specifying a local IP address and subnet mask for the tun interface. In this mode, each connected client is assigned a unique IP address, and it is compatible with Windows systems as well. P2P: Implement a point-to-point topology where the client's tun interface remote endpoint consistently points to the server's tun interface local endpoint. This configuration assigns a single IP address to each connected client. Utilize this mode exclusively when none of the connected clients operate on Windows systems."
```

---

I've separated the tooltip translation into distinct elements to allow styling without formatting raw HTML code from the translations, should be ok now!